CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 0.4.1 (October 21st, 2020)
+
+### Fixed
+
+* Remove package dependency on the synchronous DB drivers (#256)
+
```

Comment:
Are we okay with bumping to `0.5.0`? I think this qualifies as an entry for a "Changed" section, rather than "Fixed". That way people that are currently pinned to `0.4.*` will be fine.

I'm not sure I understood the broken build issue with these direct dependencies, though, but I think sticking closer to SemVer is a good idea here?

Reply:
Besides, we have to bump the version, otherwise it does deploy to pypi and many people continue suffering.

Thanks for the approval, I will carry on with the release now.

Reply:
Makes sense!